vector/src/main/res/layout/item_positive_destrutive_buttons.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:gravity="center_horizontal"
+    android:orientation="horizontal"
+    tools:visibility="visible">
+
+    <Button
+        android:id="@+id/destructive_button"
+        style="@style/Widget.Vector.Button.Destructive"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="16dp"
+        android:text="@string/action_decline" />
+
+    <Button
+        android:id="@+id/positive_button"
+        style="@style/Widget.Vector.Button.Positive"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/action_accept" />
```

Comment:
Since there is default values for the text here, maybe use more generic wording, like "Yes" and "No".